src/models/BibDetails.ts

```diff
@@ -113,6 +117,100 @@ export default class BibDetailsModel {
     }
   }
 
+  get groupedNotes() {
+    const note = this.bib?.note?.length ? this.bib.note : null
+
+    // Make sure we have at least one note
+    if (note && Array.isArray(note)) {
+      // Group notes by noteType:
+      return (
+        note
+          // Make sure all notes are blanknodes:
+          .filter((note) => typeof note === "object")
+          .reduce((groups, note) => {
+            const noteType = this.getNoteType(note)
+            if (!groups[noteType]) {
+              groups[noteType] = []
+            }
+            groups[noteType].push(note.prefLabel)
+            return groups
+          }, {})
+      )
+    }
+  }
+
+  /**
+   * getNoteType(note)
+   * Construct label for a note by adding the word 'Note'
+   */
+  getNoteType = (note: Note) => {
+    const type = note.noteType || ""
+    return type.toLowerCase().includes("note") ? type : `${type} (note)`
+  }
+
+  /**
+   * Combines properties from matching (i.e. parallel) elements as necessary
+   * Right now, this is only needed to add the 'noteType' in case of parallel notes
+   */
+  combineMatching(primaryValue: string, parallelValue: string | Note) {
```

Comment:
Small nit, but I'm thinking combineMatching, interleaveParallel and matchParallel's could use more descriptive fields? Even if it's just interleaveParallelFields etc?